recipes/app1.rb

```diff
@@ -18,7 +18,17 @@
 
 include_recipe 'osl-app::default'
 
-node.default['users'] = %w(openid-staging openid-production)
+users = search('users', '*:*')
+
+users_manage 'app1' do
+  users users
+end
+
+users.each do |u|
+  directory "/home/#{u[:username] || u[:id]}" do
+    mode '02750'
```

Comment:
Is this what you were asking about the other day about what permission to set for this? Seems like this is something we should be able to configure in the data bag or have a better default in the upstream cookbook?